packages/ramldoc-generator/src/main.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+
+function generateRaml(options) {
+  const resolvedSourcePath = path.resolve(options.src);
+
+  // Check if the spec file exists.
+  if (!fs.existsSync(resolvedSourcePath)) {
+    throw new Error(chalk.red.bold(`"${resolvedSourcePath}" does not exist`));
+  }
+
+  generateRamlIfSpecPath({
+    ...options,
+    src: resolvedSourcePath,
+  });
+}
+
+function generateRamlIfSpecPath(options) {
+  const resolvedDestinationPath = path.resolve(options.dest);
+
+  console.log(
+    `RAML docs will be generated in this directory: ${chalk.blue(
+      resolvedDestinationPath
+    )}`
+  );
+
+  const jarFile = path.resolve(__dirname, '../jar/rmf-codegen.jar');
```

Comment:
@davifantasia can you explain the process of getting the JAR file? We should also document this...

cc @nkuehn

Reply:
I don't know how to go about documenting that process:

- is it a part of the ReadMe?
- what kind of title would it have if documented?
- is the documentation targeted at users or someone who is just interested in how the package works?
- do you have an existing structure for documenting something like this?
- other things you want to add.

Reply:
You can add a file `CONTRIBUTING.md` and describe there where to get the JAR file. This is usually targeted to people that wants to contribute to the development of the package.
Or just add a section to the `README.md`.